pkg/store/sandbox/sandbox.go

```diff
@@ -31,7 +31,9 @@ type Sandbox struct {
 	Metadata
 	// Containerd sandbox container
 	Container containerd.Container
-	// TODO(random-liu): Add cni network namespace client.
+	// CNI network namespace client
+	//TODO(Abhi): Should we handle multi network scenario
```

Comment:
@abhinandanpb For now, I believe the answer is no. :) When you do `PodSandboxStatus`, we only expect 1 IP to be returned.